### ISSUE TITLE
volume candle  in lightweight chart

### ISSUE THREAD

**Issue by vishnuc** [tradingview/lightweight-charts#1744]
Hi , I can see we can create plugins , can we create volume candle where each bar in price scale has different width , is it possible ? I read somewhere all width must be same...

**Comment by SlicedSilver** [comment]
The distance between the centers of data points must remain constant. You can only partially create volume candles by varying the widths of the candles. However, narrower candles will have larger whitespace around them. It is not possible to have different widths without altering the gaps between the candles.

Hopefully this explanation makes sense.

**Comment by vishnuc** [comment]
Yea I can understand , Even with custom plugins I wont be able to create ? Like how tradingview original achieved

**Comment by SlicedSilver** [comment]
Yes, you won't be able to achieve the same appearance that is possible on tradingview.com with LWC at the moment.

It has been discussed internally, and we are considering including it in a future release but it won't be part of the next major `v5` but could be part of a `v5.1` release for next year. Up to this point there hasn't been much demand for this feature.

**Comment by vishnuc** [comment]
HI , ok please consider for 5.1 , variable bar width + plugin creation will give us more flexibilty.
